pkg/sql/logictest/tests/5node/5node_test: TestLogic_ranges failed

pkg/sql/logictest/tests/5node/5node_test.TestLogic_ranges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8489176?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8489176?buildTab=artifacts#/) on master @ [502acb0d8241bcf128abd3699852be5e1c610f07](https://github.com/cockroachdb/cockroach/commits/502acb0d8241bcf128abd3699852be5e1c610f07):


```
=== RUN   TestLogic_ranges
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c1120e87cda9b48ae97a6489658fb611/logTestLogic_ranges1153574240
    test_log_scope.go:79: use -show-logs to present logs inline
[10:10:48] setting distsql_workmem='100832B';
=== CONT  TestLogic_ranges
    logic.go:4201: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c1120e87cda9b48ae97a6489658fb611/logTestLogic_ranges1153574240
--- FAIL: TestLogic_ranges (5.72s)
=== RUN   TestLogic_ranges/show_range_table_for_row_columns
    logic.go:2913: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1986/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges:589: SHOW RANGE FROM TABLE tbl_for_row FOR ROW (0)
        expected:
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
            
        but found (query options: "colnames") :
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        95        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
[10:10:52] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1986/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges with config 5node: 98 tests, 1 failures
[10:10:52] --- total progress: 98 statements
--- total: 98 tests, 1 failures
    --- FAIL: TestLogic_ranges/show_range_table_for_row_columns (0.03s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_ranges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-23959

pkg/sql/logictest/tests/5node/5node_test.TestLogic_ranges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8503542?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8503542?buildTab=artifacts#/) on master @ [5c38cfb6ab518ac2b8354c3a2a238ff3cfc88844](https://github.com/cockroachdb/cockroach/commits/5c38cfb6ab518ac2b8354c3a2a238ff3cfc88844):


```
=== RUN   TestLogic_ranges
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c1120e87cda9b48ae97a6489658fb611/logTestLogic_ranges2192739132
    test_log_scope.go:79: use -show-logs to present logs inline
[09:50:25] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2604/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges: 43 statements
=== CONT  TestLogic_ranges
    logic.go:4201: -- test log scope end --
--- FAIL: TestLogic_ranges (7.11s)
=== RUN   TestLogic_ranges/show_range_table_for_row_columns
    logic.go:2913: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2604/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges:589: SHOW RANGE FROM TABLE tbl_for_row FOR ROW (0)
        expected:
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
            
        but found (query options: "colnames") :
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        95        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
[09:50:27] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2604/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges with config 5node: 98 tests, 1 failures
[09:50:28] --- total progress: 98 statements
--- total: 98 tests, 1 failures
    --- FAIL: TestLogic_ranges/show_range_table_for_row_columns (0.03s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_ranges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/logictest/tests/5node/5node_test.TestLogic_ranges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8507022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8507022?buildTab=artifacts#/) on master @ [5c38cfb6ab518ac2b8354c3a2a238ff3cfc88844](https://github.com/cockroachdb/cockroach/commits/5c38cfb6ab518ac2b8354c3a2a238ff3cfc88844):


```
=== RUN   TestLogic_ranges
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c0293f6737fd646500a86258f12122b3/logTestLogic_ranges2034709603
    test_log_scope.go:79: use -show-logs to present logs inline
[09:44:15] setting distsql_workmem='22623B';
[09:44:17] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7775/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges: 36 statements
=== CONT  TestLogic_ranges
    logic.go:4201: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c0293f6737fd646500a86258f12122b3/logTestLogic_ranges2034709603
--- FAIL: TestLogic_ranges (8.46s)
=== RUN   TestLogic_ranges/show_range_table_for_row_columns
    logic.go:2913: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7775/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges:589: SHOW RANGE FROM TABLE tbl_for_row FOR ROW (0)
        expected:
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
            
        but found (query options: "colnames") :
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        95        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
[09:44:20] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7775/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges with config 5node: 98 tests, 1 failures
[09:44:21] --- total progress: 98 statements
--- total: 98 tests, 1 failures
    --- FAIL: TestLogic_ranges/show_range_table_for_row_columns (0.03s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_ranges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/logictest/tests/5node/5node_test.TestLogic_ranges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8524085?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8524085?buildTab=artifacts#/) on master @ [639decf058c78f0afe0dba613d60670628c92e81](https://github.com/cockroachdb/cockroach/commits/639decf058c78f0afe0dba613d60670628c92e81):


```
=== RUN   TestLogic_ranges
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c0293f6737fd646500a86258f12122b3/logTestLogic_ranges2851341361
    test_log_scope.go:79: use -show-logs to present logs inline
[08:52:45] setting distsql_workmem='62812B';
[08:52:47] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1438/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges: 44 statements
=== CONT  TestLogic_ranges
    logic.go:4201: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c0293f6737fd646500a86258f12122b3/logTestLogic_ranges2851341361
--- FAIL: TestLogic_ranges (6.42s)
=== RUN   TestLogic_ranges/show_range_table_for_row_columns
    logic.go:2913: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1438/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges:589: SHOW RANGE FROM TABLE tbl_for_row FOR ROW (0)
        expected:
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
            
        but found (query options: "colnames") :
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        95        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
[08:52:49] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1438/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges with config 5node: 98 tests, 1 failures
[08:52:50] --- total progress: 98 statements
--- total: 98 tests, 1 failures
    --- FAIL: TestLogic_ranges/show_range_table_for_row_columns (0.04s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_ranges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/logictest/tests/5node/5node_test.TestLogic_ranges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8520704?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8520704?buildTab=artifacts#/) on master @ [ac656036cf7e23408915532c433e995977eff899](https://github.com/cockroachdb/cockroach/commits/ac656036cf7e23408915532c433e995977eff899):


```
=== RUN   TestLogic_ranges
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c1120e87cda9b48ae97a6489658fb611/logTestLogic_ranges3020908743
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestLogic_ranges
    logic.go:4201: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c1120e87cda9b48ae97a6489658fb611/logTestLogic_ranges3020908743
--- FAIL: TestLogic_ranges (5.28s)
=== RUN   TestLogic_ranges/show_range_table_for_row_columns
    logic.go:2913: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2175/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges:589: SHOW RANGE FROM TABLE tbl_for_row FOR ROW (0)
        expected:
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
            
        but found (query options: "colnames") :
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        95        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
[10:17:15] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2175/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges with config 5node: 98 tests, 1 failures
[10:17:15] --- total progress: 98 statements
--- total: 98 tests, 1 failures
    --- FAIL: TestLogic_ranges/show_range_table_for_row_columns (0.02s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_ranges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/logictest/tests/5node/5node_test.TestLogic_ranges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_UnitTests_BazelUnitTests/8540709?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_UnitTests_BazelUnitTests/8540709?buildTab=artifacts#/) on master @ [da247ff22039f4579e2b043656ba561e71876115](https://github.com/cockroachdb/cockroach/commits/da247ff22039f4579e2b043656ba561e71876115):


```
=== RUN   TestLogic_ranges
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/a2566248d14bc1c9b0195e875c18c745/logTestLogic_ranges2607161785
    test_log_scope.go:79: use -show-logs to present logs inline
[07:29:59] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/8232/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges: 31 statements
=== CONT  TestLogic_ranges
    logic.go:4201: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/a2566248d14bc1c9b0195e875c18c745/logTestLogic_ranges2607161785
--- FAIL: TestLogic_ranges (10.47s)
=== RUN   TestLogic_ranges/show_range_table_for_row_columns
    logic.go:2913: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/8232/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges:589: SHOW RANGE FROM TABLE tbl_for_row FOR ROW (0)
        expected:
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
            
        but found (query options: "colnames") :
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        95        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
[07:30:03] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/8232/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges with config 5node: 98 tests, 1 failures
[07:30:05] --- total progress: 98 statements
--- total: 98 tests, 1 failures
    --- FAIL: TestLogic_ranges/show_range_table_for_row_columns (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_ranges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/logictest/tests/5node/5node_test.TestLogic_ranges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8541983?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8541983?buildTab=artifacts#/) on master @ [0fbaa27b619c385a67057000bf5a3e59e2e7b2b6](https://github.com/cockroachdb/cockroach/commits/0fbaa27b619c385a67057000bf5a3e59e2e7b2b6):


```
=== RUN   TestLogic_ranges
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c0293f6737fd646500a86258f12122b3/logTestLogic_ranges4973467
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestLogic_ranges
    logic.go:4201: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c0293f6737fd646500a86258f12122b3/logTestLogic_ranges4973467
--- FAIL: TestLogic_ranges (6.43s)
=== RUN   TestLogic_ranges/show_range_index_for_row_columns
    logic.go:2913: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/8740/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges:603: SHOW RANGE FROM INDEX tbl_with_idx_for_row@idx FOR ROW (NULL, 0)
        expected:
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
            
        but found (query options: "colnames") :
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        95        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
[14:26:12] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/8740/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges with config 5node: 99 tests, 1 failures
[14:26:13] --- total progress: 99 statements
--- total: 99 tests, 1 failures
    --- FAIL: TestLogic_ranges/show_range_index_for_row_columns (0.03s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_ranges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/logictest/tests/5node/5node_test.TestLogic_ranges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8549679?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8549679?buildTab=artifacts#/) on master @ [dc3224221324dcc4336861232003f2ec1aac4f9c](https://github.com/cockroachdb/cockroach/commits/dc3224221324dcc4336861232003f2ec1aac4f9c):


```
=== RUN   TestLogic_ranges
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c0293f6737fd646500a86258f12122b3/logTestLogic_ranges2325774704
    test_log_scope.go:79: use -show-logs to present logs inline
[01:46:44] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/8263/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges: 44 statements
=== CONT  TestLogic_ranges
    logic.go:4097: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c0293f6737fd646500a86258f12122b3/logTestLogic_ranges2325774704
--- FAIL: TestLogic_ranges (7.53s)
=== RUN   TestLogic_ranges/show_range_index_for_row_columns
    logic.go:2809: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/8263/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges:603: SHOW RANGE FROM INDEX tbl_with_idx_for_row@idx FOR ROW (NULL, 0)
        expected:
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
            
        but found (query options: "colnames") :
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        95        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
[01:46:46] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/8263/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges with config 5node: 99 tests, 1 failures
[01:46:47] --- total progress: 99 statements
--- total: 99 tests, 1 failures
    --- FAIL: TestLogic_ranges/show_range_index_for_row_columns (0.03s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_ranges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/logictest/tests/5node/5node_test.TestLogic_ranges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8554045?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8554045?buildTab=artifacts#/) on master @ [c4257c934858dcdb54cec514c1d5642d4992d5c2](https://github.com/cockroachdb/cockroach/commits/c4257c934858dcdb54cec514c1d5642d4992d5c2):


```
=== RUN   TestLogic_ranges
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c1120e87cda9b48ae97a6489658fb611/logTestLogic_ranges448097813
    test_log_scope.go:79: use -show-logs to present logs inline
[11:00:49] setting distsql_workmem='71048B';
=== CONT  TestLogic_ranges
    logic.go:4170: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c1120e87cda9b48ae97a6489658fb611/logTestLogic_ranges448097813
--- FAIL: TestLogic_ranges (5.62s)
=== RUN   TestLogic_ranges/show_range_index_for_row_columns
    logic.go:2881: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2725/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges:603: SHOW RANGE FROM INDEX tbl_with_idx_for_row@idx FOR ROW (NULL, 0)
        expected:
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
            
        but found (query options: "colnames") :
            start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
            <before:/Table/130/1/"\x80">  …        95        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
[11:00:52] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2725/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/ranges with config 5node: 99 tests, 1 failures
[11:00:53] --- total progress: 99 statements
--- total: 99 tests, 1 failures
    --- FAIL: TestLogic_ranges/show_range_index_for_row_columns (0.03s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_ranges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/logictest/testdata/logic_test/ranges
+++ b/pkg/sql/logictest/testdata/logic_test/ranges
@@ -598,26 +598,25 @@ subtest show_range_table_for_row_columns
 statement ok
 CREATE TABLE tbl_for_row(i INT PRIMARY KEY);
 
-# TODO(#96136): Re-enable this test after making it deterministic.
-# query TTIITTTTT colnames
-# SHOW RANGE FROM TABLE tbl_for_row FOR ROW (0)
-# ----
-# start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
-# <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
+# Ignore non-deterministic range_id.
+query TT_ITTTTT colnames
+SHOW RANGE FROM TABLE tbl_for_row FOR ROW (0)
+----
+start_key                     end_key       range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
+<before:/Table/130/1/"\x80">  <after:/Max>  _         1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
 
 subtest end
 
 subtest show_range_index_for_row_columns
 
+# Ignore non-deterministic range_id.
 statement ok
 CREATE TABLE tbl_with_idx_for_row(i INT, INDEX idx (i));
 
-# TODO(#96136): Re-enable this test after making it deterministic.
-# query TTIITTTTT colnames
-# query TTIITTTTT colnames
-# SHOW RANGE FROM INDEX tbl_with_idx_for_row@idx FOR ROW (NULL, 0)
-# ----
-# start_key                     end_key  range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
-# <before:/Table/130/1/"\x80">  …        94        1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
+query TT_ITTTTT colnames
+SHOW RANGE FROM INDEX tbl_with_idx_for_row@idx FOR ROW (NULL, 0)
+----
+start_key                     end_key       range_id  lease_holder  lease_holder_locality  replicas  replica_localities      voting_replicas  non_voting_replicas
+<before:/Table/130/1/"\x80">  <after:/Max>  _         1             region=test,dc=dc1     {1}       {"region=test,dc=dc1"}  {1}              {}
 
 subtest end

